generators/ionic/resources/base/package.json

```
@@ -5,14 +5,14 @@
   "homepage": "https://www.jhipster.tech",
   "author": "Ionic Framework + JHipster",
   "scripts": {
-    "build": "ionic build",
+    "build": "npx ionic build",
```

Package.json already runs the binary locally if available.
No need for npx.
```suggestion
    "build": "ionic build",
```